resource-types/VSCodeRemoteDevelopment/README.md

```diff
@@ -0,0 +1,173 @@
+# Managed VS Code Remote Dev Environments
+
+The `AWSSamples::Devtools::Devinstance` resource manages an EC2 instance and additional services that make a great dev environment for the [Visual Studio Code Remote - Containers](https://code.visualstudio.com/docs/remote/containers) extension. VS Code and other IDEs like Cloud9 are popularizing remote dev environments, which in general offer the following benefits:
+
+- Ability to acquire computing resources (CPU, RAM, GPU...) as needed and tear them down when idle
+- Don't be restricted by the hardware limits of the local machine
+- Try different CPU architectures and hardware variants with minimal effort (e.g. ARM)
+- Enjoy a consistent developer experience from any device
+- Easily recreate the dev environment with automation if something went terribly wrong
+
+The `Devinstance` resource supports this in the following ways:
+
+- **The developer can choose from a wide variety of instance types**, depending on the requirements (currently, only instances with hibernate support are allowed, although technically anything should work)
+
+- **Dev environment as code:** With [VS Code Remote Containers]((https://code.visualstudio.com/docs/remote/containers)), the developer connects to the instance via VS Code, which builds and starts a container defined by a `Dockerfile` in the developer's local project directory. The instance is preconfigured with Docker to allow VS Code to use it as a remote environment.
+
+- **The instance is cattle, but your data is persistent:** The developer can update the `Devinstance` resource to change the instance type, while not losing any data. This is possible because a persistent EBS volume is used to store the developer's workspace that is mounted into the Docker container. This volume is automatically moved between instances and is protected from deletion (will remain even after the resource is deleted).
+
+- **The size of the EBS volume can be increased at any time** by updating the resource (currently triggers the recreation of the instance, which might feel like overkill but is a nice way to stop all processes before changing volume properties). The filesystem size is extendend automatically.
+
+- **The instance has a stable hostname, even throughout recreation**, so the configuration in VS Code (the SSH URI) always stays the same. 
+
+## Installing the resource type
+
+This resource type is built with the [community Typescript plugin for CloudFormation](https://github.com/eduardomourar/cloudformation-cli-typescript-plugin):
```

Comment:
You can remove this, TypsScript is officially supported

Reply:
changed to official link and removed "community"